sdk/core/azure-core-test/src/main/java/com/azure/core/test/models/NetworkCallError.java

```diff
@@ -12,12 +13,14 @@
  * during the pipeline and deserialize them back into their actual throwable class when running in playback mode.
  */
 public class NetworkCallError {
-    @JsonProperty("Throwable")
-    private Throwable throwable;
-
     @JsonProperty("ClassName")
     private String className;
 
+    @JsonProperty("ErrorMsg")
+    private String errorMsg;
+
+    private Throwable throwable;
```

Comment:
If this isn't being serialized anymore we should remove it.

Reply:
Used in default block.